bundles/media/security.rst

```diff
@@ -4,11 +4,17 @@ Security
 Separate permissions can be applied to every collection. These permissions also
 apply to the assets contained in the collection.
 
-So the `view` permission is required to see the collection itself in the
-navigation and its assets in any available view in the system.
+So in the Sulu system (representing Sulu's administration interface) the `view`
```

Comment:
Sadly I think this is not longer true as now I need `edit` permission to see the Navigation Item:

https://github.com/sulu/sulu/blob/ae5b58ea48ebd260eb762295ef483add46d697ae/src/Sulu/Bundle/MediaBundle/Admin/MediaAdmin.php#L83

Reply:
I am talking about the permissions on the collections themselves... There you don't need the edit permission to see the collections in the UI.